styles.css

```diff
@@ -8,6 +8,17 @@ body {
   font-family: "Poppins", Arial, Helvetica, sans-serif;
 }
 
```

Comment:
[OPTIONAL]
I would suggest adding `margin: 0` to the `*`, so we are setting all elements to have zero margins, thus making them look the same in all browsers. Notice that you have a small margin on the left, right, and bottom of the contact section.
```CSS
* {
  margin: 0;
  box-sizing: border-box;
}
```